test/controllers/teacher_controller_test.exs

```diff
@@ -66,4 +101,44 @@ defmodule CoursePlanner.TeacherControllerTest do
     conn = get conn, teacher_path(conn, :new)
     assert html_response(conn, 200) =~ "New teacher"
   end
+
```

Comment:
those don't test the controller itself, could you move them to `test/lib/course_planner/teachers_test.exs`?